sample_data/egypt_by_governorate_since_2009/egypt_by_governorate_since_2009.md

```diff
@@ -0,0 +1,12 @@
+# Sources
+
+## egypt_by_governorate_since_2009.geojson
+Runfola D, Anderson A, Baier H, Crittenden M, Dowker E, Fuhrig S, et al. (2020) 
+geoBoundaries: A global database of political administrative boundaries. 
+PLoS ONE 15(4): e0231866. https://doi.org/10.1371/journal.pone.0231866. 
+Downloaded from: https://github.com/wmgeolab/geoBoundaries/tree/main/releaseData/gbOpen/EGY/ADM1 on 4 July 2022.
+
+## egypt_population_2017.csv
+Source: Central Agency for Public Mobilization and Statistics. 2017 Population, Housing, and Establishments Census. Downloaded 30 June 2022 from https://www.capmas.gov.eg/Admin/News/PressRelease/2018530104533_899-e.pdf.
```

Comment:
Are you sure this is the correct link? I cannot seem to find the data in the CSV in that link.